page-connect.php

```diff
@@ -0,0 +1,65 @@
+		<p>ownCloud Proxy is both an app inside ownCloud and a service. The ownCloud Proxy app is installed in ownCloud and configured in the ownCloud admin panel. From the ownCloud Proxy app, an admin can sign up for an ownCloud Proxy account with one of the supported providers. Once the account is confirmed, the ownCloud app initiates and maintains a secure connection to the ownCloud Proxy service. The service in turn provides a recognizable URL on the web that, when accessed via ownCloud clients, connects users back to their ownCloud server residing behind their router on a network at home or at work. Once the server is connected, simply download the desktop sync client from ownCloud.org, or get the ownCloud apps from the Apple App Store or Google Play, and get started.</p>
+	</div>
+</div>
+<div class="row">
+        <div class="col-md-12">
+                Learn more in the ownCloud <a href="<?php echo $DOCUMENTATION_ADMIN; ?>TBD">administrator documentation</a> and in the questions and answers section below.
+        </div>
+</div>
+
+
+<div class="row">
+	<div class="col-md-12">
+        <h2>Questions and answers</h2>
+        </div>
+        <div class="col-md-12">
+        <h3>How about security?</h3>
+        <p>All of the security associated with ownCloud are at work when you use the ownCloud Proxy service. ownCloud is still run by you, on your computer. You can monitor what is happening, manage accounts, check the logs, use your own SSL certificates, enable automatic updates and more. The appliance pre-configures much of ownCloud for you. All you have to do is start the appliance, complete the setup and log into the web URL it provides. The ownCloud browser wizard at the provided URL will step you through logging in for the first time. Of course, as you configure accounts for others to share on your ownCloud, we recommend you use secure and complex passwords for all accounts.</p>
+        <p>However, we want to be completely transparent about the nature of ownCloud Proxy. The ownCloud Proxy app is distributed and the source code is in the ownCloud repository for review in Github. The ownCloud Proxy service is provided by PageKite. All files transferred through the ownCloud Proxy service do pass through its data centers, and for a brief time are readable as the ownCloud Proxy service transfers incoming requests from SSL to SSH on the way to the private ownCloud Server. In theory, if someone wanted to, all files transferred to and from devices using the ownCloud Proxy service could – for that brief instant of time that they are being moved from the SSL terminated connection to the SSH tunnel in memory – read by someone with ownCloud Proxy service server access. While the risk is small, and our terms and conditions cover this situation, it is something we wanted to be clear about. However, there is no way for the ownCloud Proxy service to access your data on your private server, it can only broker a connection with your private server.</p>
```

Comment:
With pagekite it's the pagekite protocol instead SSH. See https://pagekite.net/wiki/Floss/PageKiteProtocol/